### PR TITLE
Sensor updates

### PR DESCRIPTION
**Pull request by rhysfaultless-cpr**

### CODE REVIEW

File: docs/sensors/lidar_3d/velodyne_puck hi_res.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+| Description                 | Value                |
+| :-------------------------- | :------------------- |
+| Mass                        | 830 g                |
+| Voltage, Minimum            | 9 V                  |
+| Voltage, Maximum            | 18 V                 |
+| Voltage, Sensor             | 12 V                 |
+| Power, Average              | 8 W                  |
+| Data                        | Ethernet, 100 Mbit/s |
+| Range, Minimum              |                      |
```
Comment (jhiggins-cpr) [inline docs/sensors/lidar_3d/velodyne_puck hi_res.mdx]: Is the 'Minimum' unknown or N/A?

**Review by jhiggins-cpr — APPROVED** [review]

File: docs/sensors/lidar_3d/velodyne_puck hi_res.mdx (hunk 1 above)
Comment (rhysfaultless-cpr) [reply docs/sensors/lidar_3d/velodyne_puck hi_res.mdx]: Velodyne does not state a minimum in their datasheets or manuals.
The ROS driver disregards data that is closer than 0.9 metres:
https://github.com/ros-drivers/velodyne/blob/7c2f199f552ecb9f58a3c30734681042b1b99f82/velodyne_pointcloud/launch/transform_nodelet.launch#L11

Should we add 0.9 metres to our table's cell?

Comment (jhiggins-cpr) [reply docs/sensors/lidar_3d/velodyne_puck hi_res.mdx]: I think we should add something to the cell. I'm ok with any of:

- N/A
- 0.9 m

**Review by jhiggins-cpr — APPROVED** [review]

File: docs/sensors/lidar_3d/velodyne_puck hi_res.mdx (hunk 1 above)
Comment (rhysfaultless-cpr) [reply docs/sensors/lidar_3d/velodyne_puck hi_res.mdx]: I updated our 4 Velodyne pages to say to 0.9 m.

**Pull request merged**